### ISSUE TITLE
weirdness

### ISSUE THREAD

**Issue by zeke** [lxe/require-navigator#16]
I'm running version 0.2.0 of the extension. Sometimes I see weirdness like this:

![screen shot 2015-02-22 at 10 48 09 pm](https://cloud.githubusercontent.com/assets/2289/6323785/022377c8-bae5-11e4-9559-f991541311cc.png)

https://github.com/rvagg/github-webhook/blob/master/package.json

**Comment by taylorjbrennan** [comment]
Yeah, that was my fault. It should be fixed, but it hasn't been published. Maybe we can bug @lxe to get this into the Chrome Web Store.

**Comment by yuchi** [comment]
By the way, it happens every time I open a source file from the repo’s *home*.
